client/me/security-2fa-backup-codes/password-prompt.jsx

```diff
@@ -0,0 +1,68 @@
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import React from 'react';
+import FormButton from 'calypso/components/forms/form-button';
+import FormFieldset from 'calypso/components/forms/form-fieldset';
+import FormLabel from 'calypso/components/forms/form-label';
+import FormSettingExplanation from 'calypso/components/forms/form-setting-explanation';
+import FormTextInput from 'calypso/components/forms/form-text-input';
+
+class Security2faBackupCodesPasswordPromt extends React.Component {
```

Comment:
This looks like a typo:

```suggestion
class Security2faBackupCodesPasswordPrompt extends React.Component {
```